src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -57,32 +57,34 @@ the main body of the message as well as a quote.
 				<div v-else-if="showJoinCallButton" class="message-body__main__text call-started">
 					<NcRichText :text="message"
 						:arguments="richParameters"
-						:autolink="true"
+						autolink
 						:reference-limit="0" />
 					<CallButton />
 				</div>
-				<div v-else-if="showResultsButton" class="message-body__main__text system-message">
+				<div v-else-if="showResultsButton || isSystemMessage" class="message-body__main__text system-message">
 					<NcRichText :text="message"
 						:arguments="richParameters"
-						:autolink="true"
+						autolink
 						:reference-limit="0" />
 					<!-- Displays only the "see results" button with the results modal -->
-					<Poll :id="messageParameters.poll.id"
+					<Poll v-if="showResultsButton"
+						:id="messageParameters.poll.id"
 						:poll-name="messageParameters.poll.name"
 						:token="token"
-						:show-as-button="true" />
+						show-as-button />
 				</div>
 				<div v-else-if="isDeletedMessage" class="message-body__main__text deleted-message">
 					<NcRichText :text="message"
 						:arguments="richParameters"
-						:autolink="true"
+						autolink
 						:reference-limit="0" />
 				</div>
-				<div v-else class="message-body__main__text" :class="{'system-message': isSystemMessage}">
+				<div v-else class="message-body__main__text message-body__main__text--markdown">
 					<Quote v-if="parent" :parent-id="parent" v-bind="quote" />
 					<NcRichText :text="message"
 						:arguments="richParameters"
-						:autolink="true"
+						autolink
+						use-markdown
```

Comment:
Should only be done when the markdown property is passed.
(Also reference limit shoulkd be 0 in that case)